server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
@puellanivis If you search "Metadata" in golang, you can also find tons of places which use Metadata (e.g., https://github.com/golang/crypto/blob/c197bcf24cde29d3f73c7b4ac6fd41f4384e8af6/ssh/connection.go#L24). The same thing happens in google internal go RPC lib when this concept is needed. This is really a non-issue... To me, meta is fine and I prefer metadata.